bin/GameIndex.dbf

```diff
@@ -1263,7 +1278,8 @@ Serial = SCES-50354
 Name   = Klonoa 2 - Lunatea's Veil
 Region = PAL-M5
 Compat = 5
-eeClampMode = 3		//Objects needed appear in wrong places without it
+eeClampMode = 3 //Objects needed appear in wrong places without it
```

Comment:
Small nitpick, maybe add space between // and Objects so it's consistent with the other changes.
Could do the rest with the others that need it as well.